.spelling

```diff
@@ -61,4 +61,7 @@ vcc
 VSCode
 W3Schools
 Webots
+www.python.org
```

Comment:
These items don't appear to be used in this PR?

Reply:
they're not, but my local copy of the spellchecker was flagging them up